src/js/actions/tools.js

```diff
@@ -65,6 +65,8 @@ define(function (require, exports) {
      */
     var _currentTransformPolicyID = null;
 
+    var directSelectToolID = "superselectVector";
```

Comment:
@iwehrman Should this string be stored in `stores/tools.js` and then retrieved from there or is this ok? If the latter, can you explain why?